content/blog/2024-05-16.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Version 2.0.0 released!"
+date: 2024-05-15T11:00:00+02:00
```

Comment:
date not in phase with the filename

Reply:
yeah, that is however really a minor thing without any side effects

Reply:
usually this attribute is used to hide/show the blog post. So depending when the blogpost should be visible it can have an impact

Reply:
we thought to make it visible now but I switched now to the 16th so this shouldn'T be a concern any more.